src/Sarif/Schemata/sarif-schema.json

```diff
@@ -14,7 +14,7 @@
 
     "version": {
       "description": "The SARIF format version of this log file.",
-      "enum": [ "2.0.0-csd.2.beta.2019-01-24" ]
+      "enum": [ "2.0.0-csd.2.beta.2019-02-20" ]
```

Comment:
>02- [](start = 39, length = 3)

um, gee, thanks, jschema validation. this was an important omission to detect. :)

Reply:
Ack, now I'm so confused. You pushed a schema versioned with 01-24 to schemastore, but you pushed test files versioned with 02-20. I don't know what to believe ;-)

---
In reply to: [259604231](https://github.com/Microsoft/sarif-sdk/pull/1307#discussion_r259604231) [](ancestors = 259604231)

Reply:
I think I understand now. **PLEASE** see my email to confirm my understanding.

---
In reply to: [259648735](https://github.com/Microsoft/sarif-sdk/pull/1307#discussion_r259648735) [](ancestors = 259648735,259604231)